App/src/pages/CourseBrowser.tsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import Select from "react-select";
+import { Box, Button, Container, FormLabel, Heading, Stack } from "@chakra-ui/react";
+
+interface SubjectItem {
+    value: string;
+    label: string;
+}
+
+const SUBJECT_ITEMS: Array<SubjectItem> = [
+    { value: "COMP", label: "COMP" },
+    { value: "SOEN", label: "SOEN" },
+    { value: "ENGR", label: "ENGR" },
+];
+
+interface CatalogItem {
+    value: string;
+    label: string;
+}
+
+const CATALOG_ITEMS: Array<CatalogItem> = [
+    { value: "201", label: "201" },
+    { value: "301", label: "301" },
+    { value: "401", label: "401" },
+];
+
+const CourseSelector1 = () => <Select options={SUBJECT_ITEMS} />;
+const CourseSelector2 = () => <Select options={CATALOG_ITEMS} />;
```

Comment:
Is there a reason those are functions instead of adding them straight into the return value of the function (on lines 51 and 55 respectively)? 
Also heads up, you might want to create a `useState` to capture the value of the selector(s) but that can wait till the backend is ready.

Reply:
My reference example was using methods, but you are right I don't see another need to have them now.

Reply:
Let's indeed wait for the backend but thanks for the heads up.